docs/guides/admin/docs/modules/subtitles.md

```diff
@@ -0,0 +1,79 @@
+Subtitles
+=========
+
+This document records guidelines and conventions on how subtitles should be handled in Opencast.
+
+<div class=warn>
+These guidelines are new to Opencast 14. Not all parts of Opencast may yet adhere to them! 
+</div>
+
+Subtitles are expected to be in [WebVTT format](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API). Subtitles
+in other formats are expected to be converted to WebVTT prior to uploading them to Opencast.
+
+## Storage
+
+In general, subtitles are stored as tracks in an event, much like audio and video streams. Subtitles stored this way
+are assumed to be related to audio and video tracks in the event.
+
+### Flavor
+
+For the flavor of a subtitle track, `captions` is recommended for the main flavor, while the sub-flavor should express
+the processing state (e.g. `source`, `work`, `delivery`). In the case of multiple subtitle tracks, they should all
+be stored in the same flavor, while being optionally differentiated by [tags](#tags).
+
+### Tags
+
+Tags are used to hold additional information about individual subtitle tracks. All tags are optional and components
+should work without them being present, falling back to generic displays, or not showing information at all.
+
+The following is a list of tags that Opencast components may recognize:
+- `lang:<language>`
+	- `<language>` represents a language code of the form TODO: WHICH LANGUAGE CODE?
```

Comment:
I assume these Todos should be done before we merge this?

Reply:
Thanks for the reminder.

Regarding language codes: I don't have a solid opinion on which to use. Do you? I should probably bring this is up in the Technical Meeting.

Regarding no major language: Might just have to ignore that for now and cross that bridge when we come to it?

Reply:
> Regarding language codes: I don't have a solid opinion on which to use. Do you?

I think I prefer including the region in the language code, so `en-GB` instead of `en`. Most people commenting on the issue also seemed to think so.

> I should probably bring this is up in the Technical Meeting.

Good idea.

> Regarding no major language: Might just have to ignore that for now and cross that bridge when we come to it?

Agreed.